workflows/deephyper_hpo/hpo_deephyper_subprocess.py

```diff
@@ -0,0 +1,149 @@
+import json
+import subprocess
+import pandas as pd
+import os
+from pathlib import Path
+import logging
+import mpi4py
+from deephyper.evaluator import Evaluator, profile
+from deephyper.evaluator.callback import TqdmCallback
+from deephyper.problem import HpProblem
+from deephyper.search.hps import CBO
+from mpi4py import MPI
+import socket
+import hpo_deephyper_params_def
+from improvelib.config.base import Config
```

Comment:
@nkoussa @adpartin I see that the base.py is longer there in the config folder.

Reply:
Thanks, I can see this comment. Will fix this after the PR is approved.

Reply:
@adpartin Please see here